kafka/conn.py

```diff
@@ -778,7 +778,8 @@ def filter(self, record):
         log.addFilter(log_filter)
 
         test_cases = [
-            ((0, 10), ApiVersionRequest[0]()),
+            ((0, 10, 1), MetadataRequest[2]([])),
```

Comment:
note to self: for any broker after 0.10 we should just use the results of ApiVersions api request to set api version support.